packages/lodestar/src/network/metadata/utils.ts

```diff
@@ -0,0 +1,36 @@
+import {IBeaconConfig, IForkInfo} from "@chainsafe/lodestar-config";
+import {Epoch, phase0} from "@chainsafe/lodestar-types";
+import {FAR_FUTURE_EPOCH} from "../../constants";
+import {IForkDigestContext} from "../../util/forkDigestContext";
+
+export function getENRForkID(
```

Comment:
This function is specific to the Metadata class and only used here. It doesn't make sense to be attached to the BeaconChain class

Reply:
I think we put that in BeaconChain because it links to the head state while the implementation below always link to current clock slot / epoch.

The difference happens when we're at altair (current slot / epoch is at altair) and we're syncing from phase0, in that case ENRForkID inside the ENR should be respective to phase0, not altair

Reply:
RIght, good pick! Fixed